### DISCUSSION TITLE
iroh v0.6.0 is out!

### DISCUSSION THREAD

**Discussion by b5** [n0-computer/iroh#1543] [Announcements]
![iroh_v0 6 0_release](https://github.com/n0-computer/iroh/assets/1154390/c13acd93-e03d-43ea-8d31-5a1497f233c2)

# The one with sync!

This release gets us out of alpha with [documents](https://iroh.computer/docs/documents), check the [release](https://github.com/n0-computer/iroh/releases/tag/v0.6.0) for detailed info 😄

**Comment by machawk1** [comment]
@b5 With today's release of 0.10, should this announcement in the Discussion tab of the repo be updated/removed?

**Reply by b5** [reply]
lol yes!
